zhaquirks/xiaomi/aqara/light_acn.py

```diff
@@ -45,6 +56,23 @@ class OppleClusterLight(XiaomiAqaraE1Cluster):
     }
 
 
+class AqaraLightT1M(XiaomiAqaraE1Cluster):
+    """Add power on behavior management for Lumi devices."""
+
+    manufacturer_id_override = 4447
```

Comment:
Does the device provide another manufacturer id in the NodeDescriptor?

Reply:
Yes, this is what is in the node descriptor
```
  "node_descriptor": {
    "logical_type": 1,
    "complex_descriptor_available": 0,
    "user_descriptor_available": 0,
    "reserved": 0,
    "aps_flags": 0,
    "frequency_band": 8,
    "mac_capability_flags": 142,
    "manufacturer_code": 4447,
    "maximum_buffer_size": 82,
    "maximum_incoming_transfer_size": 82,
    "server_mask": 11264,
    "maximum_outgoing_transfer_size": 82,
    "descriptor_capability_field": 0
  },
  ```

Reply:
Ok, since the power on attribute is already marked as `is_manufacturer_specific`, zigpy will automatically use that manufacturer id then.
So, the override can be removed from what I can see.